snuba/settings_base.py

```diff
@@ -13,7 +13,10 @@
 DATASET_MODE = 'local'
 
 # Clickhouse Options
-CLICKHOUSE_SERVER = os.environ.get('CLICKHOUSE_SERVER', 'localhost:9000')
+# TODO: Warn about using `CLICKHOUSE_SERVER`, users should use the new settings instead.
+[default_clickhouse_host, default_clickhouse_port] = os.environ.get('CLICKHOUSE_SERVER', 'localhost:9000').split(':', 1)
```

Comment:
`sentry devservices` (and the Sentry tests in CI) use this setting, so there needs to be some amount of grace period before removing it to avoid disrupting other workflows.

Reply:
ok